conformance/utils/kubernetes/apply_test.go

```diff
@@ -27,6 +28,15 @@ import (
 	"sigs.k8s.io/gateway-api/apis/v1alpha2"
 )
 
+var (
+	// These aren't used by this test but since this test is run using
+	// the same invocation as the conformance test it needs to parse the
+	// same set of flags.
+	_ = flag.String("gateway-class", "", "unused")
+	_ = flag.Bool("debug", false, "unused")
+	_ = flag.Bool("cleanup-base-resources", true, "unused")
+)
+
```

Comment:
Maybe we need a config or flags package so we don't have to repeat these in both places?